src/main/kotlin/no/nav/familie/ks/sak/statistikk/saksstatistikk/Behandlingtilstand.kt

```diff
@@ -1,28 +1,26 @@
 package no.nav.familie.ks.sak.statistikk.saksstatistikk
 
-import no.nav.familie.kontrakter.felles.tilbakekreving.Ytelsestype
 import no.nav.familie.ks.sak.kjerne.behandling.domene.BehandlingStatus
 import no.nav.familie.ks.sak.kjerne.behandling.domene.BehandlingType
+import no.nav.familie.ks.sak.kjerne.behandling.domene.Behandlingsresultat
 import no.nav.familie.ks.sak.kjerne.behandling.domene.BehandlingÅrsak
 import java.time.OffsetDateTime
 
 data class Behandlingtilstand(
     val funksjoneltTidspunkt: OffsetDateTime, // Tidspunkt for siste endring på behandlingen.
     val tekniskTidspunkt: OffsetDateTime? = null, // Tidspunktet da fagsystemet legger hendelsen på grensesnittet/topicen
     val mottattTid: OffsetDateTime? = null, // Tidspunktet da behandlingen oppstår (eks søknadstidspunkt, inntektsmelding, etc). Det er ønskelig å måle brukers opplevde ventetid. Ved elektronisk kontakt regner vi med at denne er lik registrertTid
-    val saksnummer: String,
-    val ytelseType: Ytelsestype,
+    val saksnummer: Long,
     val behandlingID: Long,
     val behandlingType: BehandlingType,
     val behandlingStatus: BehandlingStatus,
-    val behandlingResultat: String? = null, // Behandlingresultatstype,// TODO legg til når vi er ferdig med reusltat
+    val behandlingResultat: Behandlingsresultat? = null,
```

Comment:
```suggestion
    val behandlingsResultat: Behandlingsresultat? = null,
```
1-1 mellom enum og variabelnavn